README.md

```diff
@@ -16,7 +16,7 @@ We like reactive programming and with this experiment we hope to push this forwa
 
 ## Features
 
-- ✅ Works with both `ViewEngine` and `Ivy`
+- ✅ Works with both `ViewEngine` and `Ivy` (_note: if you want to use `ngx-template-streams` with **Ivy** you have to use the lastest version of `ngx-template-streams`_)
```

Comment:
Could you add an `*` after Ivy and add the note below the features?

Here's how I would change it:

```text
- ✅ Works with both `ViewEngine` and `Ivy` [\*](#ivy-compatibility-note)

...

**Notes**

[<a id="ivy-compatibility-note"></a>1] If you want to use `ngx-template-streams` with **Ivy** you have to use the **lastest** version of `ngx-template-streams`.
```